### DISCUSSION TITLE
Request for Assistance with Photon Brilliance Calculations

### DISCUSSION THREAD

**Discussion by ghost** [epochpic/epoch#821] [General]
I am currently working on a project involving the generation of high-brilliance photon beams through intense laser-plasma interactions. Specifically, I am focusing on the nonlinear Thomson scattering process to produce photon
﻿
I am seeking guidance on the most accurate methods to calculate the photon brilliance, 

Could anyone provide insights or direct me to resources that detail the calculation of photon brilliance in the context of high-intensity laser-plasma interactions?

Thank you in advance for your assistance.

**Comment by TomGoffrey** [comment]
As this is a request for research assistance, rather than a report of a code error, a request for a new feature, or a specific query on functionality, I will convert this to a discussion.
